content/en/cosign/verify.md

```diff
@@ -76,6 +76,14 @@ Verify image with local certificate and local certificate chain:
 $ cosign verify --cert cosign.crt --cert-chain chain.crt user/demo
 ```
 
+## Keyless verification using only a provided certificate chain with non-Fulcio roots
```

Comment:
We can drop keyless, the default in the docs is to use identity based signing. I’d change this instead to be:
“Verify image with user-provided trusted chain”

Reply:
done in 5ea5c75